packages/lodestar/test/unit/chain/validation/block.test.ts

```diff
@@ -35,6 +36,9 @@ describe("gossip block validation", function () {
     chain.forkChoice = forkChoice;
     regen = chain.regen = sinon.createStubInstance(StateRegenerator);
 
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+    (chain as any).opts = {maxSkipSlots};
```

Comment:
You can cast to the full class, which should allow editing. Better than any

Reply:
Unfortunately on the full class ie `BeaconChain`, `opts` is a readonly value, so casting it to the full class will not allow editing

Reply:
Please keep trying other strategies before casting to `any`, for example

```ts
((chain as Pick<BeaconChain, "opts">) as {opts: BeaconChain["opts"]}).opts = {maxSkipSlots}
```

Or what about just
```ts
chain.opts.maxSkipSlots = maxSkipSlots
```

Just keep iterating until you find the strongest chain of casting that will not compile if something changes